src/Controller/Controller.php

```diff
@@ -368,7 +383,6 @@ public function setRequest(Request $request)
     {
         $this->request = $request;
         $this->plugin = isset($request->params['plugin']) ? $request->params['plugin'] : null;
-        $this->view = isset($request->params['action']) ? $request->params['action'] : null;
```

Comment:
Won't this mean the property won't be set at the beginning of a controller action anymore?